src/community/importer-fgdb/src/main/resources/git.properties

```diff
@@ -0,0 +1,15 @@
+#Generated by Git-Commit-Id-Plugin
```

Comment:
Is this file really meant to be committed?

Reply:
Nope, it isn't, leftover from the repo it was copied from. I've removed it.